addons/web/static/src/legacy/js/core/dom.js

```diff
@@ -545,11 +545,14 @@ var dom = {
      * @param {number} [options.forcedOffset]
      *      offset used instead of the automatic one (extraOffset will be
      *      ignored too)
+     * @param {JQuery} [options.$scrollable] the $element to scroll
      * @return {Promise}
      */
     scrollTo(el, options = {}) {
         const $el = $(el);
-        const $scrollable = $el.parent().closestScrollable();
+        const $scrollable = options.$scrollable || $el.parent().closestScrollable();
```

Comment:
A bit strange to have a scroll To taking an element as parameter, but target is coming from options ... I understand the global issue, but seems a bit coding by side-effects... I don't have any better proposal as this is not my main scope, just wondering a bit.